lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,325 @@
+            "currency": {
+                "coding": [
+                    {
+                        "system": "urn:iso:std:iso:4217",
+                        "code": "USD",
+                        "display": "United States Dollar",
+                        "userSelected": true
+                    }
+                ],
+                "text": "United States Dollar"
+            }
+        }
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-amount
+
+- name: financial-transaction-allocation
+  required: 'Yes'
+  cardinality: 1..*
+  type: BackboneElement
+  description: How the FinancialTransaction should be applied.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation",
+      "extension": [
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-target",
+              "valueReference": {
+                  "reference": "Encounter/31363178"
+              }
+          },
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-amount",
+              "valueMoney": {
+                  "value": "400000",
+                  "currency": {
+                      "coding": [
+                          {
+                              "system": "urn:iso:std:iso:4217",
+                              "code": "USD",
+                              "display": "United States Dollar",
+                              "userSelected": true
+                          }
+                      ],
+                      "text": "United States Dollar"
+                  }
+              }
+          }
+      ]
+    }
+  note: Only the first allocation element will be used when creating the resource for this initial release.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation
+
+- name: financial-transaction-allocation-target
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference
+  description: The specific target the FinancialTransaction should be posted to.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-target",
+      "valueReference": {
+            "reference": "Encounter/31363178"
+      		}
+    }
+  note: Encounter will be the only posting level target that will be supported for this initial release.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-target
+
+- name: financial-transaction-allocation-amount
+  required: 'Yes'
+  cardinality: 1..1
+  type: Money
+  description: The amount of the Financial Transaction that is to be allocated to the specific target.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation-amount",
```

Comment:
amount

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4824eee1801e3185a37a5b6f312294e683160a0e#diff-26870e1b94fc9955e130f3f940dbe2e7R263

Reply:
Looks done

+1

Reply:
Ohhh woops I remember what the change is. This just needs to be amount

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/06664b8f9cc72c310bfbc416f91cbb6df08a74b5#diff-26870e1b94fc9955e130f3f940dbe2e7L184